lia-ui/src/main/java/com/lithium/community/android/ui/components/adapters/LiConversationAdapter.java

```diff
@@ -74,12 +80,18 @@
  * This class is responsible for displaying the message thread.
  */
 public class LiConversationAdapter extends LiBaseRecyclerAdapter {
+
+    public interface ActionInProgressListener {
```

Comment:
why is the inner class public here ?

Reply:
thats an interface

Reply:
Is the behavior tied to the adapter specifically? Would prefer in a separate file otherwise as opposed to inner.

Reply:
Yes.